README.md

```diff
@@ -125,11 +125,11 @@ $ kubectl-kruise migrate --help
    * [x] status
    * [x] pause
    * [x] resume
-   * [ ] restart
+   * [x] restart
```

Comment:
I don't see any code about `rollout restart` in this PR?

Reply:
`rollout restart for cloneSet` has been developed in last PR and I forgot to add it in ReadMe, so add it to ReadMe in this PR.